xarray/tests/test_convert.py

```diff
@@ -0,0 +1,28 @@
+import numpy as np
```

Comment:
I agree that a new module for conversion tests makes sense, but we currently some of these tests in `test_dataarray.py`: https://github.com/pydata/xarray/blob/9c8005937556211a8bf28a946744da3768846c5a/xarray/tests/test_dataarray.py#L2917-L3083

Could I trouble you to move these over into this file?